gas-estimation/src/eth_node.rs

```diff
@@ -1,18 +1,21 @@
 //! Ethereum node `GasPriceEstimating` implementation.
 
 use super::GasPriceEstimating;
-use crate::contracts::Web3;
 use anyhow::Result;
-use pricegraph::num;
 use std::time::Duration;
+use web3::{Transport, Web3};
 
 #[async_trait::async_trait]
-impl GasPriceEstimating for Web3 {
+impl<T> GasPriceEstimating for Web3<T>
+where
+    T: Transport + Send + Sync,
+    <T as Transport>::Out: Send,
+{
     async fn estimate_with_limits(&self, _gas_limit: f64, _time_limit: Duration) -> Result<f64> {
         self.eth()
             .gas_price()
             .await
             .map_err(From::from)
-            .map(num::u256_to_f64)
+            .map(|r| r.to_f64_lossy())
```

Comment:
Right... so you have already done this.

Reply:
There are still other callsites in services_core